apps/src/applab/dropletConfig.js

```diff
@@ -8,6 +10,19 @@ var COLOR_RED = '#F78183';
 var COLOR_CYAN = '#4DD0E1';
 var COLOR_YELLOW = '#FFF176';
 
+/**
+ * Generate a list of screen ids for our setScreen dropdown
+ */
+function getScreenIds() {
+  var ret = $(".screen").map(function () {
+    return '"' + this.id + '"';
+  });
+
+  // Convert from jQuery's array-like object to a true array
+  return $.makeArray(ret);
```

Comment:
Hmm `ret` is already an array for me when I tried this out in Chrome.  Is it needed for another browser?